app/models/tax_return.rb

```diff
@@ -67,7 +67,7 @@ class TaxReturn < ApplicationRecord
   def qualifying_dependents
     raise StandardError, "Qualifying dependent logic is only valid for 2020.  Define new rules for #{year}." unless year == 2020
 
-    intake.dependents.reject(&:soft_deleted_at).filter { |d| d.qualifying_child_2020? || d.qualifying_relative_2020? }
+    intake.dependents.filter { |d| d.qualifying_child_2020? || d.qualifying_relative_2020? }
```

Comment:
The eip eligibility methods are currently using qualifying_dependents (so that they could take advantage of the soft deletion for the hot fix). I'm now wondering if we should revert to having those method call intake.dependents since they call the qualification logic in those methods, so we're kind of looping through the same set of dependents twice.